tests/conformance/other/nothing/test.yaml

```diff
@@ -0,0 +1,4 @@
+description: |
+    - node "three" is present since there is no othe requirement.
```

Comment:
I dont understand this. Node "three" is present since is has no conditions assigned.